client/src/components/SignupForm.js

```diff
@@ -20,8 +20,8 @@ const styles = theme => ({
 });
 
 class SignupForm extends Component {
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
```

Comment:
Why change this?

Reply:
my ide was yelling at me about it saying how it should match the parent constructor

Reply:
my ide was yelling at me about it, saying it should match the parent constructor (I'm going to assume this is one of those instances where javascript is quite happy to ignore the fact that we didn't put in a parameter)